src/assets/styles/variables.css

```diff
@@ -8,7 +8,15 @@ html {
   /* Some elements are stubborn so we need to force the color there */
   --txt-colour: var(--colour-ti-base);
 
-  --transition: all 250ms ease-out;
-  --transition-transform: 0.15s transform ease-in-out;
-  --transition-opacity: 0.15s opacity ease-in-out;
+  --transition: none;
+  --transition-transform: none;
+  --transition-opacity: none;
```

Comment:
I think we don't need those, since this is the fallback value. Shall we delete them or do you think this increases code readability?